lib/features/survey/lauguage_selection_page.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/material.dart';
```

Comment:
for launcher icon errors, try to run `flutter pub run flutter_launcher_icons:main` on the terminal.